src/storage/storage.ts

```diff
@@ -913,4 +914,32 @@ export class Storage implements IStorage {
     const accessData = await this._getValidAccessData();
     return (accessData.walletFlags & WALLET_FLAGS.HARDWARE) > 0;
   }
+
+  /**
+   * Return if the nano contract is registered for the given address based on ncKey.
+   * @param ncKey Pair address:ncId concatenated.
+   * @returns `true` if registered and `false` otherwise.
+   * @async
+   */
+  async isNanoContractRegistered(ncKey: string): Promise<boolean> {
+    return this.store.isNanoContractRegistered(ncKey);
+  }
+
+  /**
+   * Get nano contract data.
+   * @param ncKey Pair address:ncId registered.
+   * @returns An instance of Nano Contract data.
+   */
+  async getNanoContract(ncKey: string): Promise<INcData | null> {
+    return this.store.getNanoContract(ncKey);
+  }
+
+  /**
+   * Register nano contract data instance.
+   * @param ncKey Address and nano contract id concatenated.
+   * @param ncData Nano Contract basic information.
+   */
+  async registerNanoContract(ncKey: string, ncValue: INcData): Promise<void> {
```

Comment:
Isn't it missing the unregister method?

Reply:
Yes. I will change it.

Reply:
Added here https://github.com/HathorNetwork/hathor-wallet-lib/pull/614/commits/d0e5fb4abc067dc94642e189eba1ebbc3c486f00